openpype/pipeline/delivery.py

```diff
@@ -120,7 +120,9 @@ def deliver_single_file(
     anatomy_data,
     format_dict,
     report_items,
-    log
+    log,
+    renumber_frame=False,
```

Comment:
I'm not sure if it makes sense to have renumbering logic in deliver single file? We don't have any frame information about the file and it can easily try to change e.g. shot number instead of frame.

Reply:
> I'm not sure if it makes sense to have renumbering logic in deliver single file? We don't have any frame information about the file and it can easily try to change e.g. shot number instead of frame.
When I first tried to implement  with `deliver_sequences` into delivery widget, it's not working as expected.  @antirotor told me that the related function is deprecated in Openpype 3 so I used `deliver_single_file` instead.